code/game/jobs/job/job.dm

```diff
@@ -79,22 +79,22 @@
 	if(!account_allowed || (H.mind && H.mind.initial_account))
 		return
 
-	var/loyalty = 1
+	var/econ_status = 1
 	if(H.client)
-		switch(H.client.prefs.nanotrasen_relation)
-			if(COMPANY_LOYAL)        loyalty = 1.30
-			if(COMPANY_SUPPORTATIVE) loyalty = 1.15
-			if(COMPANY_NEUTRAL)      loyalty = 1
-			if(COMPANY_SKEPTICAL)    loyalty = 0.85
-			if(COMPANY_OPPOSED)      loyalty = 0.70
+		switch(H.client.prefs.economic_status)
+			if(ECONOMICALLY_WEALTHY)        econ_status = 1.50
+			if(ECONOMICALLY_WELLOFF) 		econ_status = 1.25
```

Comment:
The higher levels should be kept as is imho. 1.50 is too big of a bonus.

Reply:
Resolved, however statuses below average are still more poor than they used to be, though not so drastically.